.pre-commit-config.frontend.yaml.jinja

```
@@ -0,0 +1,72 @@
+{% set eslint_files = '\\.(js|ts|vue)$' %}
+{% set eslint_exclude = '^nuxt-template-overrides/.*$' %}
+
+{% extends '.pre-commit-config.yaml.jinja' %}
+
+{% block note %}
+{{ super() }}
+# - https://github.com/WordPress/openverse/blob/main/.pre-commit-config.frontend.yaml.jinja
+{%- endblock %}
+
+{#==================================
+Blocks must be indented by 2 spaces.
```

I was confused by this for a second (which blocks?), does this suggestion correctly clarify it?

```suggestion
YAML blocks must be indented by 2 spaces.
```

---

I meant 'repo' blocks because there are indented inside the `repos` key in the final YAML.

---

Clarified in 0f48093.

---

Thank you :pray: